app/view_models/auction_show_view_model.rb

```diff
@@ -66,9 +70,17 @@ def eligibility_label
     end
   end
 
+  def bid_form_prompt
```

Comment:
Is this something we should delegate to Rules objects? Or is that polymorphism overkill?